recipes/light-and-complex-resources.md

```diff
@@ -0,0 +1,90 @@
+---
+description: I need a light version and a complete version of the same resource
+complexity: 1
+---
+
+# Light and complex resources
+
+## Use-case
+
+On my e-commerce API, I have an entity called Product, representing a product sold on the API.\
+A Product can have many attributes, but the API Clients usually don't need all of them at the same time.
+
+A single resource `GET /products/:id` would be too big for most usages, but all the attributes are needed at a step of my workflow.
+
+How can I build an API that meets all these requirements?
+
+## Recipe 1: expose several resources with real business meaning
+
+If it is too big, it is probably because your resource tries to cover multiple business cases: display a list of products as a search result, the details of a product, the technical info, the prices, the stocks, etc.\
```

Comment:
```suggestion
If your ressource is too big, it is probably because your resource tries to cover multiple business cases: display a list of products as a search result, details of a product, technical info, prices, stocks, etc.\
```